crypto/fipsmodule/FIPS.md

```diff
@@ -50,8 +50,6 @@ FIPS requires that RNG state be zeroed when the process exits. In order to imple
 
 By default, entropy is sourced using a "Passive" method where the specific entropy source depends on the OE. Seeding and reseeding material for the DRBG is sourced from the specific entropy source.
 
-In FIPS mode, each of the entropy sources is subject to a 10× overread. That is, when *n* bytes of entropy are needed, *10n* bytes will be read from the entropy source and XORed down to *n* bytes.
```

Comment:
Based on https://github.com/aws/aws-lc/blob/7962f75d9e836cd5ad84beed3385add9157c8a30/crypto/fipsmodule/rand/rand.c#L141 I think we can say that, when Jitter is used, we do a 3x oversampling (the Jitter default setting).

Reply:
I've added a comment about it and clarified that the CPU Jitter is not the default entropy source.